assignment1/src/com/assignment1/InputOutput.java

```diff
@@ -0,0 +1,96 @@
+package com.assignment1;
+import java.util.Scanner;
+
+public class InputOutput {
+	Scanner s = new Scanner(System.in);
+	
+	public Item inputItem() {
+		Item item = new Item();
```

Comment:
Input to object creation was asked to be done using command line arguments. Could you implement that ??

Reply:
Sure.